js/src/BrushSelector.ts

```diff
@@ -296,7 +296,7 @@ export class BrushSelector extends BrushMixinXYSelector {
         } else {
             const extent = [[selected_x[0], selected_y[0]],
                           [selected_x[1], selected_y[1]]];
-            this.brush.extent(extent);
+            this.brush.move(this.d3el, extent);
```

Comment:
@martinRenou this one might also be needed in #1202 I think when you update selected_x and selected_y from the kernel, this misbehaved. I should have better explained this in this PR.

Reply:
Ok I will apply it to my PR.